### ISSUE TITLE
Make user name editable

### ISSUE THREAD

**Issue by Swati4star** [project-travel-mate/Travel-Mate#157]
- On the profile page, there is a field to show user name
- Make that field ediatble ( add an edit icon be the side of username )
- When user clicks on save or ( ✔️ icon), send the new name to server

Use the following endpoint: 
POST api to `update-user-details`
parameters : `firstname`, `lastname`

**Comment by brijeshshah13** [comment]
@Swati4star I'm taking up this one!

**Comment by Swati4star** [comment]
Yes. Keep one field only for the name (what we have right now)
Allow user to enter full name there. Split the name by space. Keep first string as first name & other strings as last name.

**Comment by brijeshshah13** [comment]
@Swati4star Can you please post a sample UI? I can't picture the location of the field where the name will be displayed and where it can edited.

**Comment by Swati4star** [comment]
Do it in the profile page only. (that you created)

**Comment by brijeshshah13** [comment]
@Swati4star Yeah, i got that. You mean there should be a option to edit the existing field which we use to display name, correct?

**Comment by Swati4star** [comment]
Yes.
